test/Scripts.Integration.Test/build-project.ps1

```diff
@@ -15,14 +15,22 @@ $outputPath = "$NewProjectBuildPath/$(GetTestAppName $buildMethod)"
 Write-Host "Executing ${buildMethod}:"
 $unityArgs = @("-batchmode", "-projectPath ", "$NewProjectPath", "-executeMethod", $buildMethod , "-buildPath", $outputPath, "-quit")
 
-if ($CheckSymbols)
+# The iOS symbol upload gets validated as part of the actual Xcode build
+if($Platform -eq "iOS" -And $CheckSymbols)
 {
+    $unityArgs += "-uploadSymbols"
+    RunUnityCustom $unityPath $unityArgs
+}
+elseif ($CheckSymbols)
+{
+    $unityArgs += "-uploadSymbols"
```

Comment:
I guess this is related to my previous question. Why do we need this, though? It worked before without explicitly handling iOS here.